### DISCUSSION TITLE
service to show weather alerts on a map

### DISCUSSION THREAD

**Discussion by eadmaster** [bareboat-necessities/lysmarine_gen#278] [Q&A]
Do you know a web or radio service to receive weather alerts and display them on a map?

I know about [Navtex](https://en.wikipedia.org/wiki/NAVTEX), but i does not provide parseable GPS coordinates for each alert.

Basically something like this, but for maritime use:
https://severeweather.wmo.int/v2/

**Comment by mgrouch** [comment]
I know OpenCPN can show DSC alerts on charts. 

There are requests for other alerts:
https://github.com/OpenCPN/OpenCPN/issues/2815

https://github.com/OpenCPN/OpenCPN/issues/2814

Also VHF radios display weather alerts but not sure how or if they report them into NMEA network.
